### DISCUSSION TITLE
PVE Post Install script broken

### DISCUSSION THREAD

**Discussion by thom2804** [community-scripts/ProxmoxVE#422] [Q&A] [answered]
Today I installed my very first Proxmox host after using VMWare ESXi for a while giving this a shot, I already found the GitHub and helper scripts page, however it seems like the Post Install script is broken, after executing it, it returns the following:
```
 ✗ This version of Proxmox Virtual Environment is not supported
Requires Proxmox Virtual Environment Version 8.0 or later.
```
I downloaded and installed the latest version 8.3.0, which apparently only released a few hours ago, I guess the script needs to be updated to support the new version?

Also any tips and tricks are welcome as im only just starting to learn Proxmox 😃

**Comment by MickLesk** [comment, marked as answer]
Yes, 8.3 is available since today.I'm not that fast :smile:

**Reply by thom2804** [reply]
No worries offcourse, atleast I am sure now that I didn't do something wrong on my end, I will hold off a bit with the script until the updated version is available.
Thanks in advance!
